src/Effectful/Log.hs

```diff
@@ -44,18 +45,19 @@ runLog
   -> Eff (Log : es) a
   -- ^ The computation to run.
   -> Eff es a
-runLog component logger maxLogLevel = reinterpret reader $ \env -> \case
-  LogMessageOp level message data_ -> do
-    time <- liftIO getCurrentTime
-    logEnv <- ask
-    liftIO $ logMessageIO logEnv time level message data_
-  LocalData data_ action -> localSeqUnlift env $ \unlift -> do
-    (`local` unlift action) $ \logEnv -> logEnv { leData = data_ ++ leData logEnv }
-  LocalDomain domain action -> localSeqUnlift env $ \unlift -> do
-    (`local` unlift action) $ \logEnv -> logEnv { leDomain = leDomain logEnv ++ [domain] }
-  LocalMaxLogLevel level action -> localSeqUnlift env $ \unlift -> do
-    (`local` unlift action) $ \logEnv -> logEnv { leMaxLogLevel = level }
-  GetLoggerEnv -> ask
+runLog component logger maxLogLevel =
+  reinterpret reader (\env -> \case
```

Comment:
Hmm, I think putting the handler in the where clause will be better for readability - right now it's somewhat hard to see the "handle" stuff at the end.